main.ipynb

```diff
@@ -0,0 +1,128 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import the different classes (which import other packages)\n",
+    "from src.in_out_class import InClass, OutClass\n",
+    "from src.numerics import NumericsClass\n",
+    "from src.statistics import StatisticsClass\n",
+    "\n",
+    "# Set constants\n",
+    "thres_stat = 1.0e-5\n",
+    "thres_num = 1.0e-5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Read in data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = InClass()\n",
+    "\n",
+    "data.read_statistics_data()\n",
+    "\n",
+    "data.read_numerics_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run Statistic Analysis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "statistics = StatisticsClass(data, thres_stat)\n",
```

Comment:
Do we want StatisticsClass to only accept "data" as argument? Based on the requirements I thought we explicitly pass the different data parts to it. If we just want to use data as argument the __init__ method of StatisticsClass would need to be changed.

Reply:
Changed the StatisticsClass so that it accepts data as argument

Reply:
thx